src/ontology/nbo-edit.owl

```diff
@@ -1053,6 +1053,7 @@ Declaration(Class(obo:UBERON_0002101))
 Declaration(Class(obo:UBERON_0002104))
 Declaration(Class(obo:UBERON_0002105))
 Declaration(Class(obo:UBERON_0005725))
+Declaration(Class(<http://purl.obolibrary.org/obo/nbo.owl/NBO_0070027>))
```

Comment:
Something went wront with your protege config.

1. Open nbo on this branch
1. Click on the new class
2. Click (in menu) on "refactor"-->"rename entity"
3. Click "show full IRI"
4. Change to `http://purl.obolibrary.org/obo/NBO_0070027` (without the nbo.owl)
5. Safe, commit, push